airbyte-integrations/connectors/source-exchange-rates/source_exchange_rates/source.py

```diff
@@ -38,26 +38,26 @@ class ExchangeRates(HttpStream):
     date_field_name = "date"
 
     # HttpStream related fields
-    url_base = "https://api.ratesapi.io/"
+    url_base = "http://api.exchangeratesapi.io/"
```

Comment:
I'm not sure why can't we move `v1` part here

Reply:
Sorry, I just changed the previous `api/` to `v1/` but you are right, will move `v1/` to `url_base`.

Reply:
Done.